src/app/ui/AppProvider/NavigationProvider.tsx

```diff
@@ -26,9 +26,15 @@ const getLinking = ():
   | LinkingOptions<ReactNavigation.RootParamList>
   | undefined => {
   if (!DEEP_LINK_PREFIX) {
-    Logger.error(
-      `[${LOGGER_MODULE_NAME}] No deep link prefix found, deep linking will not work.`,
-    );
+    if (__DEV__) {
```

Comment:
Please avoid chaining if-statements. Therefore, please move this condition to the upper-level condition (`if (!DEEP_LINK_PREFIX)`)